tornjak-frontend/src/components/agent-create-join-token.tsx

```diff
@@ -114,6 +115,11 @@ class CreateJoinToken extends Component<CreateJoinTokenProp, CreateJoinTokenStat
   onSubmit(e: { preventDefault: () => void; }): void {
     e.preventDefault()
 
+    if (this.state.ttl === "") {
```

Comment:
let's check for integer. I don't think `2h` or `-1` would be supported. I will check that.

Reply:
The field is an integer spinbox, so it only allows for integers already. It also threw backend errors when a negative was attempted, but the response was not very informative so I added an explicit check for this.